.github/workflows/dockerpublish.yml

```diff
@@ -89,7 +89,7 @@ jobs:
           [[ "${{ github.ref }}" == "refs/tags/"* ]] && VERSION=$(echo $VERSION | sed -e 's/^v//')
 
           # Use Docker `latest` tag convention
-          [ "$VERSION" == "master" ] && VERSION=0.0.1-SNAPSHOT
+          [ "$VERSION" == "master" ] && VERSION=1.0.0-SNAPSHOT
```

Comment:
I do not think this is enough. 
If I understand the logic of the script correctly, the branch in github is release/v1.0.0 - not "master" anymore, so the VERSION will not be set to 1.0.0.-SNAPSHOT, as intended for the release/v1.0.0 branch.